docs/how-to-guides/running-feast-in-production.md

```diff
@@ -203,7 +203,7 @@ feature_vector = fs.get_online_features(
 ).to_dict()
 ```
 
-### 4.2. Deploy Feast feature servers on Kubernetes
+### 4.2. Deploy Feast feature servers on Kubernetes (Deprecated replaced by [feast-operator](../../infra/feast-operator/README.md))
 
```

Comment:
I think it is better to mention the deprecation notice in this page also
https://github.com/feast-dev/feast/tree/master/infra/charts/feast-feature-server

Reply:
my opinion is it should be  `helm operator` not helm chart which keep supporting the installation for feast feature servers for community.

Reply:
> my opinion is it should be `helm operator` not helm chart which keep supporting the installation for feast feature servers for community.

Thats what I though, waiting for a replay from the others as well

Reply:
> I think it is better to mention the deprecation notice in this page. beginning with some attention emojis to grab attention. https://github.com/feast-dev/feast/tree/master/infra/charts/feast-feature-server

So at the moment I don't think this should be done